charts/cloudserver-front/templates/dashboard.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: sample-grafana-dashboard
```

Comment:
This needs to be based on the `release` name, like any other object created by a chart.